src/libraries/System.Runtime/tests/System.Runtime.Tests/System/TimeZoneInfoTests.cs

```diff
@@ -2776,6 +2775,15 @@ public static void NoBackwardTimeZones()
                 throw new SkipTestException("This test won't work on API level < 26");
             }
 
+            // Clear cached data to always ensure predictable results
+            TimeZoneInfo.ClearCachedData();
+            if (SupportLegacyTimeZoneNames)
+            {
+                // Get legacy time zone "UCT" and ensure is not included in list returned by TimeZoneInfo.GetSystemTimeZones
+                // to ensure no duplicates display names as we should have "UTC" in the list.
+                TimeZoneInfo uct = TimeZoneInfo.FindSystemTimeZoneById("UCT");
```

Comment:
I'm missing it... how does getting this timezone ensure it's not included in the list?

Reply:
Because if it is in the list the test will fail. We'll get two time zones (UTC and UCT) with the same Display names and this will cause `Assert.Equal(tzCollection.Count, tzDisplayNames.Count);` to fail